contracts/connection-router/src/events.rs

```diff
@@ -109,46 +101,45 @@ impl From<ChainUnfrozen> for Event {
     }
 }
 
+impl From<NewMessage> for Vec<Attribute> {
+    fn from(other: NewMessage) -> Self {
+        vec![
+            ("id", other.id_on_chain).into(),
```

Comment:
Why `other.id_on_chain` here instead of `other.global_id()`?

Reply:
As far as I can tell, the global ID is only important for us internally to guarantee uniqueness, it's not a property txs usually care about. Also, the chain data would be duplicated, that's more places data can get mangled. So I'd rather have this be the json representation of the message. In fact, it would be ideal if we could do something like `serde_json::to_string(message).to_attributes()`, but that would require parsing the string